### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by ManuelHentschel**
This PR improves the documentation of individual functions, adds two vignettes with contents from the vscode-R-debugger repo, and updates license/description.

@renkun-ken @tdeenes I added your names to the contributor list in the DESCRIPTION file, is the spelling etc. ok (and do you want to appear there)?

### CODE REVIEW

**Comment by renkun-ken** [comment]
Thanks for the nice work! Glad to appear in the contributor list.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+    person(
+        given = "Manuel",
+        family = "Hentschel",
+        role = c("aut", "cre"),
+        email = "Manuel.Hentschel@outlook.com"
+    ),
+    person(
+        given = "Kun",
+        family = "Ren",
+        role = "ctb"
```
Comment (renkun-ken) [inline DESCRIPTION:14]: Would be nice if my email is added like https://github.com/REditorSupport/languageserver/blob/master/DESCRIPTION#L14.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+    ),
+    person(
```
Comment (tdeenes) [inline DESCRIPTION:17]: Can you please have it this way?
```
    person(
      given = "Dénes", 
      family = "Tóth", 
      role = "ctb",
      email = "toth.denes@kogentum.hu", 
      comment = c(ORCID = "0000-0003-4262-3217")
   )
```

Comment (tdeenes) [reply DESCRIPTION:17]: And it is a honor to be mentioned here, thank you @ManuelHentschel .

**Pull request merged**